_shared/project/tox.ini

```diff
@@ -69,12 +70,13 @@ deps =
     format,checkformatting: black
     format,checkformatting: isort
     lint: toml
-    lint: pylint
+    lint: pylint>=3.0.0
```

Comment:
For tox to recreate the venvs of Python packages so that pylint 3.0 gets installed